src/ziggurat/init.clj

```diff
@@ -88,18 +93,36 @@
   (mount/stop #'server/server)
   (stop-messaging))
 
+(defn start-batch-consumer [args]
+  (-> (mount/only #{#'thread-pool/executor-thread-pool
```

Comment:
`executor-thread-pool` I don't like this name. It has repeated `thread-pool` and just doesn't tell me what is this namespace intended for.